src/common/Throttle.h

```diff
@@ -338,17 +338,29 @@ class TokenBucketThrottle {
   struct Bucket {
     CephContext *cct;
     const std::string name;
-    std::atomic<uint64_t> remain = { 0 }, max = { 0 };
 
-    Bucket(CephContext *cct, const std::string& n, uint64_t m)
-      : cct(cct), name(n),
-	remain(m), max(m)
-    {
-    }
+    std::atomic<uint64_t> cir = { 0 };
```

Comment:
Nit: I think all these acronym variables should be spelled out (i.e. ```committed_burst_size```) for clarity